release-doc-in-process/all_vf.TTL

```diff
@@ -28,10 +28,14 @@ vf:Action  a                 owl:Class ;
 
 vf:AgentRelationshipRole  a  owl:Class ;
         rdfs:label           "vf:AgentRelationshipRole" ;
-        rdfs:subClassOf      rdf:Property ;
         vs:term_status       "unstable" ;
         rdfs:comment         "A relationship role defining the kind of association one agent can have with another." .
 
+vf:AgentRoleBehavior  a      owl:Class ;
```

Comment:
Made it up.  Will include picture below, confusing without that.  And very interested in feedback, since this we haven't talked much about the differences of AgentRelationshipRole and Action.  AgentRelationshipRole needs to be user defined, and I think that is best in data rather than vocabulary definitions.  So I created AgentRoleBehavior as a place to have the things we do need to define in the vocabulary.

Reply:
For me the AgentRelationshipRole was very clear, but AgentRoleBehavior is not to me. Maybe some examples of individuals would help.

Reply:
First cut individuals:
```
vf:member a                 owl:NamedIndividual ,
                            vf:AgentRoleBehavior ;
        rdfs:comment        "An agent who is part of a group and participates in the group. Members can be people or organizations." ;
        rdfs:label          "member" ;
        vs:term_status      "unstable" .

vf:tradingPartner a         owl:NamedIndividual ,
                            vf:AgentRoleBehavior ;
        rdfs:comment        "An agent who trades with with another agent, often with a supplier or customer relationship." ;
        rdfs:label          "trading partner" ;
        vs:term_status      "unstable" .

vf:partOf a                 owl:NamedIndividual ,
                            vf:AgentRoleBehavior ;
        rdfs:comment        "An organization that is part of or a sub-organization of another organization.  Implies that the agent is not part of any other organization." ;
        rdfs:label          "part of" ;
        vs:term_status      "unstable" .

vf:follower a               owl:NamedIndividual ,
                            vf:AgentRoleBehavior ;
        rdfs:comment        "An agent who registers to be notified and able to view activities of another agent.  This is a one-direction relationship." ;
        rdfs:label          "follower" ;
        vs:term_status      "unstable" .

vf:other a                  owl:NamedIndividual ,
                            vf:AgentRoleBehavior ;
        rdfs:comment        "Any role behavior not otherwise defined." ;
        rdfs:label          "other" ;
        vs:term_status      "unstable" .

```